.github/workflows/release.yaml

```diff
@@ -55,4 +55,16 @@ jobs:
           SPACELIFT_API_KEY_ID: ${{ secrets.SPACELIFT_API_KEY_ID }}
           SPACELIFT_API_KEY_SECRET: ${{ secrets.SPACELIFT_API_KEY_SECRET }}
         run:
-          spacectl provider create-version --type=nexus
\ No newline at end of file
+          spacectl provider create-version --type=nexus
+      
+      - name: Publish provider
+        if: startsWith(github.ref, 'refs/tags/')
+        env:
```

Comment:
🍷 I've left the vars as separate entries as I dont necessarily want the secrets exposed in other jobs _and_ it feels right to have publish and release as two separate jobs, if publish fails, it might be useful to have the draft in the UI. If not necessary, I could amalgamate the two jobs Publish and Release.